_posts/2024-04-14-blog.md

```diff
@@ -0,0 +1,87 @@
+---
+layout: post
+title: "Missing data and PCA"
+subtitle: "How much is too much?"
+authors:
+ - Thiseas C. Lamnidis
+categories: Blog
+tag: Blog
+---
+
+_by [Thiseas C. Lamnidis](https://www.eva.mpg.de/archaeogenetics/staff/thiseas-christos-lamnidis/)_
+
+## Introduction
+
+In the field of ancient human population genetics, we often analyse ancient genomes with high proportions of missing genotypes.
+This can stem from limitations of material preservation, and/or of available laboratory/sequencing technologies.
+Given that sampling remains for ancient DNA is a destructive process, we have the ethical responsibility to make the most of the data we are able to generate.
+Additionally, we need to know exactly the limitations of our data when inferring results, and present our findings fairly and compassionately.
+After all, the past breeds the present, and the interpretations we make of the past can have implications affecting present-day peoples and communities.
+
+To that end, archaeogenticists often use quality cutoff thresholds in our analyses, limiting the set of analysed individuals to ones with sufficient coverage for population genetics analyses.
+These cutoffs are often expressed in terms of the "Number of covered SNPs", i.e. the number of SNPs which do not have a missing genotype call for a given ancient individual.
+But what constitutes a sensible minimum cutoff?
+
+It is often the case that most (if not all) sampled individuals will be featured in a Principal Component Analysis (PCA), which is both a very useful visualisation tool for human aDNA data, and the source of many-a [popgen bingo](https://ancientpopgen-bingo.firebaseapp.com/?game=ancientpopgen-bingo).
+How does missing data affect the projected position of ancient individuals on a PCA plot?
+Can you trust the placement of that low coverage individual in the paper you're reviewing, and if so to what degree?
+
+Let's find out!
+
+## Experimental setup
+
+I started with a dataset of present day individuals on the Human Origins array (\~600k SNPs) <sup>[[1](http://dx.doi.org/10.1534/genetics.112.145037),[2](http://dx.doi.org/10.1038/nature13673),[3](https://doi.org/10.1038/s41559-019-0878-2)]</sup>.
+I picked a set of candidate present-day individuals (plus the 'Loschbour' hunter-gatherer) whose positions roughly spanned the area of each PCA plot.
+I then randomly added missing data to each candidate individual's genotypes at a given rate, creating 200 replicates per missingness rate, per individual.
+I then used smartpca [4](http://dx.doi.org/10.1371/journal.pgen.0020190) to run PCA and projected all replicates (`lsqproject: YES`) across all missingness rates for all candidate individuals. The original candidate individuals were not projected, with the exception of 'Loschbour', which is also the only individual with pseudohaploid genotypes in the experiment.
```

Comment:
Maybe wrap 
```suggestion
I then used smartpca ([4](http://dx.doi.org/10.1371/journal.pgen.0020190)) to run PCA and projected all replicates (`lsqproject: YES`) across all missingness rates for all candidate individuals. The original candidate individuals were not projected, with the exception of 'Loschbour', which is also the only individual with pseudohaploid genotypes in the experiment.
```

Maybe wrap citations in quotes? Renders a bit wierd otherwise...

Reply:
And no actual commands or code?

Reply:
What does pseudohaploid genotypes got to do with anything?